Lib/test/test_tarfile.py

```diff
@@ -228,6 +228,7 @@ def test_add_dir_getmember(self):
     def add_dir_and_getmember(self, name):
         with os_helper.temp_cwd():
             with tarfile.open(tmpname, 'w') as tar:
+                tar.format = tarfile.USTAR_FORMAT
```

Comment:
Why was this change to test_tarfile add_dir_and_getmember made?

It causes the test to fail when run as users with a high uid that the ancient format cannot represent.

If this format is necessary for some reason the code needs to carry comments stating why and some test should fail without it.

https://github.com/python/cpython/issues/101334

Reply:
oh, nevermind, I see this is a method specifically within a USTAR related test class. :)